espnet_model_zoo/table.csv

```diff
@@ -106,3 +106,4 @@ spgispeech,asr,Shinji Watanabe/spgispeech_asr_train_asr_conformer6_n_fft512_hop_
 spgispeech,asr,Shinji Watanabe/spgispeech_asr_train_asr_conformer6_n_fft512_hop_length256_raw_en_bpe5000_valid.acc.ave,https://zenodo.org/record/4585546/files/asr_train_asr_conformer6_n_fft512_hop_length256_raw_en_bpe5000_valid.acc.ave.zip?download=1,16000,en,,1.7.1,0.9.8,,true
 librispeech,asr,kamo-naoyuki/librispeech_asr_train_asr_conformer6_n_fft512_hop_length256_raw_en_bpe5000_scheduler_confwarmup_steps40000_optim_conflr0.0025_sp_valid.acc.ave,https://zenodo.org/record/4604066/files/asr_train_asr_conformer6_n_fft512_hop_length256_raw_en_bpe5000_scheduler_confwarmup_steps40000_optim_conflr0.0025_sp_valid.acc.ave.zip?download=1,16000,en,,1.8.0,0.9.8,2ccd176,true
 aishell,asr,Emiru Tsunoo/aishell_asr_train_asr_streaming_transformer_raw_zh_char_sp_valid.acc.ave,https://zenodo.org/record/4604023/files/asr_train_asr_streaming_transformer_raw_zh_char_sp_valid.acc.ave.zip?download=1,16000,zh,,1.4.0,0.9.7,,true
+gigaspeech,asr,Shinji Watanabe/gigaspeech_asr_train_asr_raw_en_bpe5000_valid.acc.ave,,https://zenodo.org/record/4630406/files/asr_train_asr_raw_en_bpe5000_valid.acc.ave.zip?download=1,16000,en,,1.7.1,0.9.8,,true
```

Comment:
There is an extra `,` between the model name and zenodo link, remove them should be fine then.

Reply:
Thanks! I applied it.